app/Http/Controllers/FormController.php

```diff
@@ -48,28 +51,37 @@ public function store(Request $request)
 
 	public function show(Form $form)
 	{
+
+		if ($form->show_time_to_complete == true) {
+			$form->load('formMetrics');
+			$form->time_to_complete = $form->formMetrics ? $form->formMetrics->total_time / $form->formMetrics->total_respondents : null;
+			$form->makeHidden(['formMetrics']);
+		}
+
+		$form->show_time_to_complete = (bool) $form->show_time_to_complete;
```

Comment:
```suggestion
			$form->time_to_complete = $form->show_time_to_complete ? $form->getTimetoComplete() : null;
```

Acredito que tu poderia simplificar esse dado para uma linha se:
- Retornar um simples null caso esteja desativado
- abstrair a lógica do cálculo para o seu próprio método. Isso também ajuda a refatorar esse cálculo no futuro se for preciso.